azure-mgmt-containerinstance/src/main/java/com/microsoft/azure/management/containerinstance/ContainerGroup.java

```diff
@@ -0,0 +1,729 @@
+                 *   the details of the share as part of defining the container group, then you specify how you wan
+                 *   the volume mounted within one or more of the containers in the group.
+                 *
+                 * @param volumeName the volume name as defined in the volumes of the container group
+                 * @param mountPath the local path the volume will be mounted at
+                 * @return the next stage of the definition
+                 * @throws IllegalArgumentException thrown if volumeName was not defined in the respective container group definition stage.
+                 */
+                WithAttach<ParentT> withReadOnlyVolumeMountSetting(String volumeName, String mountPath);
+
+                /**
+                 * Specifies the container group's volume to be mounted by the container instance at a specified mount path.
+                 * <p>
+                 * Mounting an Azure file share as a volume in a container is a two-step process. First, you provide
+                 *   the details of the share as part of defining the container group, then you specify how you wan
+                 *   the volume mounted within one or more of the containers in the group.
+                 *
+                 * @param volumeMountSetting the name and value pair representing volume names as defined in the volumes of the container group and the local paths the volume will be mounted at
+                 * @return the next stage of the definition
+                 * @throws IllegalArgumentException thrown if volumeName was not defined in the respective container group definition stage.
+                 */
+                WithAttach<ParentT> withReadOnlyVolumeMountSetting(Map<String, String> volumeMountSetting);
+            }
+
+            /** The final stage of the container instance definition.
+             * <p>
+             * At this stage, any remaining optional settings can be specified, or the subnet definition
+             * can be attached to the parent virtual network definition.
+             * @param <ParentT> the stage of the parent definition to return to after attaching this definition
+             */
+            interface WithAttach<ParentT> extends
+                WithCPUCoreCount<ParentT>,
+                WithMemorySize<ParentT>,
+                WithStartingCommandLine<ParentT>,
+                WithEnvironmentVariables<ParentT>,
+                WithVolumeMountSetting<ParentT>,
+                Attachable.InDefinition<ParentT> {
+            }
+
+            /**
+             * Grouping of the container group's volume definition stages.
+             */
+            interface ContainerInstanceDefinition<ParentT> extends
+                Blank<ParentT>,
+                WithImage<ParentT>,
+                WithOrWithoutPorts<ParentT>,
+                WithPortsOrAttach<ParentT>,
+//                WithCPUCoreCount<ParentT>,
```

Comment:
what's the plan here?

Reply:
none :-) a bunch of commented code that I forgot to take out.